crates/aptos/src/node/mod.rs

```diff
@@ -1608,3 +1657,48 @@ impl Time {
         Self::new(Duration::from_secs(seconds))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::{CliResult, Tool};
+    use clap::Parser;
+
+    // TODO: there have to be cleaner ways to test things. Maybe a CLI test framework?
```

Comment:
Check out `crates/aptos/e2e`. I only added it recently but this is my attempt at a CLI E2E test framework. Still working on getting it added to CI though.

Reply:
Nice. Hopefully we can integrate that into CI/CD soon 😄